### DISCUSSION TITLE
Deprecation warning for quarkus generated tasks in Gradle 9.0

### DISCUSSION THREAD

**Discussion by devr52** [quarkusio/quarkus#33487] [Community]
Tasks nativeTestRuntimeOnly, nativeTestImplementation, integrationTestRuntimeOnly, integrationTestImplementation are affected. 
In relation to: 
- https://github.com/quarkusio/quarkus/issues/23528
- https://github.com/quarkusio/quarkus/issues/22035

Will this affect the way quarkus do thing in relation to these tasks created by the quarkus specific sourcesets?? 

> The configuration integrationTestImplementation was created explicitly. This configuration name is reserved for creation by Gradle. This behavior has been deprecated. This behavior is scheduled to be removed in Gradle 9.0. Do not create a configuration with this name. Consult the upgrading guide for further information: https://docs.gradle.org/8.1.1/userguide/upgrading_version_8.html#configurations_allowed_usage

**Comment by codeyash** [comment]
This is an issue not a discusssion.

**Comment by fwelland** [comment]
This has been cooking quite a while.    With JDK17, Q 3.6.1, and Gradle 8.5 -- this issue with the explicitly created configurations still exists.      

Gradle 9, looks like it is trending towards mid 2024.   Is there any planned resoltuion for this?
